app/components/cms_horizontal_card_component/cms_horizontal_card_component.html.erb

```diff
@@ -0,0 +1,19 @@
+<%= render GovGridRowComponent.new(padding: {top: 6, bottom: 0}) do |row|  %>
+  <%= row.with_column("full") do %>
+    <%= content_tag :div, class: [
```

Comment:
I would recommend moving the class generation up to the component, makes the view cleaner to read. You could have a method called wrapper_classes, that returns an array of classes